Modules/_ctypes/cfield.c

```diff
@@ -110,8 +110,20 @@ PyCField_new_impl(PyTypeObject *type, PyObject *name, PyObject *proto,
         goto error;
     }
 
-    Py_ssize_t bit_size = NUM_BITS(size);
-    if (bit_size) {
+    if (bit_size_obj != Py_None) {
+
+        Py_ssize_t bit_size;
```

Comment:
Lint is complaining about this. I'm assuming this just needs to get moved to before the `if`